utils/compat_regexp_generator.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this script be installed? What about move this into `utils` directory of root directory?